gender_novels/corpus_gen.py

```diff
@@ -246,10 +297,8 @@ def get_country_publication(author, title):
 
 def get_country_publication_wikidata(author, title):
```

Comment:
This whole function could use more descriptive variable names (what is wp? what is clm_dict and how does it differ from clm_list?)
Also it would be great if we had some documentation on how pywikibot works for use (or a link to it). e.g. what does `clm_dict['P495']` do? Where are the country ids from?
This is not for you to do, just for someone in the data group to clear up.